### PR TITLE
Allow cross origin requests for class outcomes

### PR DESCRIPTION
**Pull request by derekprior**
API users would like to request a list of outcomes associated with the
given class (AKA subject) from an AJAX request on their class website.
To enable this, the API needs to accept requests from domains other than
the current host. We use Rack::Cors to allow get requests to the API
from any domain.

CORS: https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
```
Comment (thorncp) [inline config/initializers/cors.rb]: Other strings in this PR use double quotes, should this file also use them?

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    resource '/api/*', headers: :any, methods: %i(get)
```
Comment (mike-burns) [inline config/initializers/cors.rb]: `%i(get)` is one character *more* than `[:get]`!

File: config/initializers/cors.rb (hunk 1 above)
Comment (derekprior) [inline config/initializers/cors.rb]: Yup. Thanks. Last client used single quotes so I'm weaning myself off of it.

File: config/initializers/cors.rb (hunk 2 above)
Comment (derekprior) [inline config/initializers/cors.rb]: I'm okay with that. It's really going to pay character saving dividends if we add `post`, `patch`, `put` and `delete`. Just think of all those saved `:` and `,`!

**Pull request merged**